File: data/imported/knowledge/cells/typescript/merged/snippets/snippet_018.ts
async function testFillSettings() {
  console.log("Testing fill settings...");

  const workbook = new Workbook();
  const worksheet = workbook.worksheets[0]!;
  const style = new Style();

  style.setForegroundColor("FFFF00");
  style.setBackgroundColor("FF0000");

  const cell = worksheet.getCell2("A1");
  cell.putValue("Colored Cell");
  cell.setStyle(style);

  await workbook.save("outputfiles/test_fill_settings.xlsx");
  console.log("Saved to outputfiles/test_fill_settings.xlsx");
}

File: data/imported/knowledge/cells/go/merged/snippets/snippet_018.go
func main() {
	wb := cells_foss.NewWorkbook()
	ws := wb.Worksheets[0]

	boldStyle := cells_foss.NewStyle()
	boldStyle.Font.Bold = true
	boldStyle.Font.Size = 12

	highlightStyle := cells_foss.NewStyle()
	highlightStyle.Font.Color = "FFFFFFFF"
	highlightStyle.Font.Bold = true
	highlightStyle.Fill = &cells_foss.Fill{
		Type:  cells_foss.FillTypeSolid,
		Color: "FF4472C4",
	}

	headers := []string{"Item", "Category", "Price", "In Stock"}
	for i, h := range headers {
		ref := string(rune('A'+i)) + "1"
		ws.Cells().Set(ref, h)
		cell, _ := ws.Cells().Get(ref)
		cell.SetStyle(boldStyle)
	}

	wb.Save("outputfiles/style.xlsx")
}
